modules/ROOT/pages/configuration/file-locations.adoc

```diff
@@ -237,8 +237,9 @@ File permissions:: Read only.
 [[neo4j-products]]
 === Products
 
-Description:: The JAR files of the Neo4j products: link:https://neo4j.com/docs/graph-data-science/current/[Graph Data Science Library], link:https://neo4j.com/docs/bloom-user-guide/current/[Neo4j Bloom], and link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/genai-integrations/[GenAI plugin].
-The folder also contains a _README.txt_ file with information on enabling them.
+Description:: The JAR files of the Neo4j products as per Neo4j Edition. +
```

Comment:
This line doesn't make much sense to me.
Do the other lines need to be bullet points? 
Maybe "The JAR files in the Neo4j products folder differ by Neo4j Edition"?

Reply:
The rest of the PR looks fine though.